client/src/components/scans/scans.js

```diff
@@ -102,18 +141,81 @@ class Scans extends React.Component {
     });
   }
 
+  runScans() {
+    Store.dispatch({
+      type: toastNotificationTypes.TOAST_ADD,
+      alertType: 'error',
+      header: 'NYI',
+      message: 'Running scans is not yet implemented'
+    });
+  }
+
+  repeatScans() {
+    Store.dispatch({
+      type: toastNotificationTypes.TOAST_ADD,
+      alertType: 'error',
+      header: 'NYI',
+      message: 'Repeating scans is not yet implemented'
+    });
+  }
+
+  downloadScans() {
+    Store.dispatch({
+      type: toastNotificationTypes.TOAST_ADD,
+      alertType: 'error',
+      header: 'NYI',
+      message: 'Downloading scans is not yet implemented'
+    });
+  }
+
+  itemSelectChange(item) {
+    const { filteredItems } = this.state;
+
+    item.selected = !item.selected;
+    let selectedItems = filteredItems.filter(item => {
+      return item.selected === true;
+    });
+
+    this.setState({ selectedItems: selectedItems });
+  }
+
+  addSource() {
+    Store.dispatch({
+      type: toastNotificationTypes.TOAST_ADD,
+      alertType: 'error',
+      header: 'NYI',
+      message: 'Importing sources is not yet implemented'
+    });
+  }
+
+  importSources() {
+    Store.dispatch({
+      type: toastNotificationTypes.TOAST_ADD,
+      alertType: 'error',
+      header: 'NYI',
+      message: 'Adding sources is not yet implemented'
+    });
+  }
+
```

Comment:
Good first round!

Probably a concept we have to evolve since it's hard to visualize, combining these ```dispatches``` into grouped actions ... so in the end, maybe something like
```js
runScans()  {
  Store.dispatch({
    type: scansTypes.DO_ALL_THINGS_FOR_RUN_SCANS,
    ...
```